addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SstiBlindScanRule.java

```diff
@@ -368,4 +380,35 @@ private void sendPayloadsToMakeCallBack(String paramName, String[] commandExecPa
             }
         }
     }
+
+    AlertBuilder createAlert(String url, String param, String attack, HttpMessage msg) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_HIGH)
+                .setUri(url)
+                .setParam(param)
+                .setAttack(attack)
+                .setMessage(msg);
```

Comment:
Let the caller do this and avoid the need to create a message for the example. Same for the other scan rule.